### PR TITLE
LSNBLDR-570; handle new URL format

### PR DESCRIPTION
**Pull request by clhedrick**
I'm not committing this myself because it includes a kernel change. The change should be safe. It just changes a method so it can be called externally.

### CODE REVIEW

**Comment by ottenhoff** [comment]
looks okay to me .... @buckett or @jonespm ?

**Comment by jonespm** [comment]
It seems fine, @steveswinsburg wrote the the method originally on SAK-23587, I don't see any reason why it needs to be private.

**Comment by buckett** [comment]
Could we just hold on this for now.

**Comment by clhedrick** [comment]
Why?

On Mar 10, 2016, at 7:18 AM, Matthew Buckett <notifications@github.com<mailto:notifications@github.com>> wrote:


Could we just hold on this for now.

-

File: kernel/api/src/main/java/org/sakaiproject/content/api/ContentHostingService.java [hunk 1]
```diff
@@ -2026,4 +2026,15 @@ public void restoreResource(String id) throws PermissionException, IdUsedExcepti
...
+
+	/**
+	 * Expand macros in a URL
+	 *
+	 * @param url - string, a URL to be expanded
+	 *
+	 * @return URL with macro expansion
+	 */
+
+	public String expandMacros(String url);;
```
Comment (steveswinsburg) [inline kernel/api/src/main/java/org/sakaiproject/content/api/ContentHostingService.java]: two ;

**Comment by buckett** [comment]
I'm fine with this now (with the tidy up @steveswinsburg suggested)

**Comment by clhedrick** [comment]
as usual I can't make git do what I want. This will turn into a new PR

**Pull request closed without merging**